packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -28,13 +28,51 @@ import { ButtonActionType } from 'modules/types';
 import { formatRep, formatAttoRep } from 'utils/format-number';
 import MarketLink from 'modules/market/components/market-link/market-link';
 import { MarketProgress } from 'modules/common/progress';
-import { InfoIcon } from 'modules/common/icons';
+import { ExclamationCircle, InfoIcon, XIcon } from 'modules/common/icons';
 import ChevronFlip from 'modules/common/chevron-flip';
 
 import TooltipStyles from 'modules/common/tooltip.styles.less';
 import Styles from 'modules/reporting/common.styles.less';
 import { Getters, convertDisplayValuetoAttoValue, convertAttoValueToDisplayValue } from '@augurproject/sdk';
 
+interface DismissableNoticeProps {
+  content: JSX.Element;
+}
+
+interface DismissableNoticeState {
+  show: boolean,
+};
+
+export class DismissableNotice extends Component<DismissableNoticeProps, {}> {
+  state: DismissableNoticeState = { show: true };
```

Comment:
this is a perfect place to use `useState` <https://reactjs.org/docs/hooks-reference.html> 

convert this to a functional component not a Component

`const [show, setShow] = useState(true);`

it'll remove a lot of code.

Reply:
Thanks for the tip! Updated.